components/expenses/ExpenseSummary.js

```diff
@@ -0,0 +1,244 @@
+import { Box, Flex } from '@rebass/grid';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+
+import expenseTypes from '../../lib/constants/expenseTypes';
+import { PayoutMethodType } from '../../lib/constants/payout-method';
+import { i18nExpenseType } from '../../lib/i18n-expense';
+import i18nPayoutMethodType from '../../lib/i18n-payout-method-type';
+import Avatar from '../Avatar';
+import Container from '../Container';
+import FormattedMoneyAmount, { DEFAULT_AMOUNT_STYLES } from '../FormattedMoneyAmount';
+import PrivateInfoIcon from '../icons/PrivateInfoIcon';
+import LinkCollective from '../LinkCollective';
+import StyledHr from '../StyledHr';
+import StyledInputField from '../StyledInputField';
+import StyledTag from '../StyledTag';
+import StyledTextarea from '../StyledTextarea';
+import { H4, P, Span } from '../Text';
+import AttachmentsTotalAmount from './AttachmentsTotalAmount';
+import PayoutMethodData from './PayoutMethodData';
+
+const msg = defineMessages({
+  notesPlaceholder: {
+    id: 'ExpenseSummary.addNotesPlaceholder',
+    defaultMessage: 'Add attachments, notes, or any other important thing. ',
+  },
+});
+
+const PrivateNoteLabel = () => {
+  return (
```

Comment:
Quick question: are we enforcing scope for every function or something like this would also be acceptable?
```jsx
const PrivateNoteLabel = () => (
  <Span fontSize="Caption" color="black.700">
    ..
  </Span>
)

Reply:
Both are fine, yours is actually better for these kind of simple components.